myapp/spec/models/user_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let!(:user) { create(:user) }
+
+  shared_examples 'When it was invalid.' do
+    it 'Invalidated and returns an error message' do
+      user.send("#{column}=", val)
```

Comment:
`update_attribute`を使うと
コールバックが実行されたりreadonlyのアトリビュートだったらエラーを起こしたりするので
いいかもしれません

```suggestion
      user.update_attribute(column, val)
```

ref: http://api.rubyonrails.org/v5.2.5/classes/ActiveRecord/Persistence.html#method-i-update

Reply:
すみません。
上記でやってみたのですが、エラーで処理が止まってしまいテストが動かなくなってしまいました。

```
     Failure/Error: user.update_attribute( column.to_sym, val)

     ActiveRecord::NotNullViolation:
       Mysql2::Error: Column 'name' cannot be null
```

update_attribute! でやっても同様で
update_attributesやupdate_attributes!  だとシンボルを可変にするのが難しそうなのですが、
何か方法をご存知でしたらご教授いただけると助かります。

Reply:
すみません！
書き方がわかりました。
こちらでできました。
https://github.com/Fablic/training/pull/1173/commits/abba954962a23318013acd9621df83f3d15606f7

Reply:
すみません。
{}が不要だったので削除しました。
https://github.com/Fablic/training/pull/1173/commits/cc333186e364b01c2d734e2b8d66b4f15f0ce656

Reply:
何度もすみません。
上記だとrubocopに怒られてたので、最終的には以下のようになりました。
https://github.com/Fablic/training/pull/1173/commits/e22e462431f4228030990b0643096f9f7a94eb24

Reply:
update_attributeだとvalidation走っちゃいますね。。失礼しました
updateでも良さそうですね。